charts/tekton-apps/templates/_snippets.tpl

```diff
@@ -215,3 +214,10 @@ the way you use it:
   value: {{ .default }}
 {{- end -}}
 {{- end -}}
+
```

Comment:
Pls run pre-commit hook linters on you codes, now it seems they not used (no empty line at the end)

Reply:
Looks strange, but hook passed. I will change end of string mannually
![Uploading image.png…]()

Reply:
![image](https://github.com/saritasa-nest/saritasa-devops-helm-charts/assets/62735583/deadd707-8444-4ec5-996e-39e1361af6df)